### ISSUE TITLE
Question about the accuracy calculation in the code

### ISSUE THREAD

**Issue by mengcao327** [DeepGraphLearning/GMNN#3]
Hi Meng, when running the code I found that the calculation for accuracy seems to be reporting the maximum accuracy from the 100 epoches. I pasted the _get_accuracy_ function from semisupervised/codes/train.py here:

def get_accuracy(results):
    best_dev, acc_test = 0.0, 0.0
    for d, t in results:
        if d > best_dev:
            best_dev, acc_test = d, t
    return acc_test

However, in your paper you said that "The mean accuracy over 100 runs is reported in experiment" (Page 7), not the highest accuracy.

Could you explain this problem of which calculation method you actually used for the results reported in the paper? Thank you!

**Comment by mnqu** [comment]
Hi Meng,

Thank you for the interest in our work!
Sorry for the confusion. In the paper, by "The mean accuracy over 100 runs", we mean that we run GMNN with 100 different random seeds, as you can see at the following line:
https://github.com/DeepGraphLearning/GMNN/blob/e8a2232bca60b8b6a43e97a9c5d8121ed83780a1/semisupervised/codes/run_cora.py#L34
In other words, we execute GMNN for 100 runs, where each run has 100 epochs and has a unique random seed. For each run we compute the "highest accuracy", or more precisely, the test accuracy at the iteration where the highest validation accuracy is achieved. Then we average over these different accuracy from 100 runs as the final reported number. It doesn't mean that we average over the accuracy obtained from 100 different epochs, and we should have made it clearer in the original paper.
Hope that my reply could help.
Again, thank you for the interest and sorry for the confusion!

Best regards,
Another Meng : )

**Comment by mengcao327** [comment]
Hi Meng,

Thank you for the detailed explaination! It helps a lot and I understand now.
This work is excellent and solid. Thanks for sharing your code from which I learned a lot.

Best wishes,
Another Meng ;-)
